dask/dataframe/core.py

```diff
@@ -2844,6 +2844,74 @@ def drop(self, labels, axis=0, errors='raise'):
     def merge(self, right, how='inner', on=None, left_on=None, right_on=None,
               left_index=False, right_index=False, suffixes=('_x', '_y'),
               indicator=False, npartitions=None, shuffle=None):
+        """Merge the DataFrame with another DataFrame
+
+        This will merge the two datasets, either on the indices, a certain column
+        in each dataset or the index in one dataset and the column in another.
+
+        1. Joining on indices: In this case the divisions are
```

Comment:
This may be appropriate for a `Notes` section below the `Returns` section.

Reply:
Moved it below the `Returns` section.
Thank you for reviewing :)